generator/src/main/java/com/algorand/sdkutils/utils/Tools.java

```diff
@@ -109,4 +116,53 @@ public static String getCamelCase(String name, boolean firstCap) {
         }
         return new String(newName, 0, n);
     }
+
+    // Imports are collected and organized before printed as import statements.
+    // addImports adds a needed import class. 
+    public static void addImport(Map<String, Set<String>> imports, String imp) {
+        String key = imp.substring(0, imp.indexOf('.'));
+        if (imports.get(key) == null) {
+            imports.put(key, new TreeSet<String>());
+        }
+        imports.get(key).add(imp);
+    }
+
+    // getImports organizes the imports and returns the block of import statements
+    // The statements are unique, and organized. 
+    public static String getImports(Map<String, Set<String>> imports) {
+        StringBuilder sb = new StringBuilder();
+
+        Set<String> java = imports.get("java");
+        if (java != null) {
+            for (String imp : java) {
+                sb.append("import " + imp + ";\n");
+            }
+            if (imports.get("com") != null) {
+                sb.append("\n");
+            }
+        }
+
+        Set<String> com = imports.get("com");
+        if (com != null) {
+            for (String imp : com) {
+                sb.append("import " + imp + ";\n");
+            }
+        }
+        sb.append("\n");
+        return sb.toString();
+    }   
+    
+
+    public static BufferedWriter getFileWriter(String className, String directory)  {
+        File f = new File(directory + "/" + className + ".java");
+        f.getParentFile().mkdirs();
+        BufferedWriter bw = null;
+        try {
+            bw = new BufferedWriter(new FileWriter(f));
+        } catch (IOException e) {
```

Comment:
When it comes to exceptions, readability of the code is favored over exception handling. 
That is, exceptions are contained where they are generated, so the upstream code does not have to deal with them. 
They are also printed and execution continues. This helps debugging, since most exceptions do not impact the following processing. 
Clearly, this is not a production grade tool. Exceptions should not be expected.